cosmoz-default-tree.html

```diff
@@ -23,134 +32,125 @@
 
 	Cosmoz.DefaultTree.prototype = Object.create(Cosmoz.Tree.prototype);
 
-	Cosmoz.DefaultTree.prototype.getNodeByProperty = function (propertyName, propertyValue) {
+	/**
+	 * Returns an Array with all found Nodes.
+	 * Returns an Object with the first found Node.
```

Comment:
@nomego  ok, can we then:
drop `getNodeByProperty()` and only keep:
- searchNodes()
- getNodeByPathLocator()

as well, drop `getPathByProperty()` and only keep:
- getPathString()

Rename:
- getPath() => getPathNodes()

or is there a reason to keep them?